src/Akkling.Cluster.Sharding/ClusterSharding.fs

```diff
@@ -86,7 +111,26 @@ let entityFactoryFor (system: ActorSystem) (name: string) (props: Props<'Message
     let adjustedProps = adjustPersistentProps props
     let shardRegion = clusterSharding.Start(name, adjustedProps.ToProps(), ClusterShardingSettings.Create(system), new TypedMessageExtractor<_,_>(EntityRefs.entityRefExtractor))
     { ShardRegion = shardRegion; TypeName = name }
-
+/// <summary>
+/// Creates a shard region and returns a factory function which for a given `shardId` and `entityId` returns a <see cref="IEntityRef{T}"/> representing
+/// a serializable entity reference to a created sharded actor. This ref can be passed as message payload and will always point to a correct entity location
+/// even after rebalancing. It also offers support for remember entities
+/// </summary>
+/// <param name="system"></param>
+/// <param name="shardResolver">used for shard resolution from entity id. For remember entities support</param>
+/// <param name="name"></param>
+/// <param name="props"></param>
+let entityFactoryForEx (system: ActorSystem) (shardResolver:ShardResolver) (name: string) (props: Props<'Message>) (rememberEntities) : EntityFac<'Message> =
```

Comment:
Since we already have variant which deals with non-remembered entities (`entityFactoryFor`), maybe this should be a function that deals specifically with remembered variant eg. `rememberedEntityFactoryFor` or something like that.

Reply:
Yes if this would be the case then no need for the book parameter either.